dev/AppNotifications/AppNotificationActivatedEventArgs.h

```diff
@@ -8,7 +8,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         AppNotificationActivatedEventArgs() = default;
 
         AppNotificationActivatedEventArgs(winrt::hstring const& arguments, winrt::Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> const& userInput) : m_arguments(arguments), m_userInput(userInput) {};
-        winrt::hstring ActivationArgs() { return m_arguments; };
+        winrt::hstring Argument() { return m_arguments; };
```

Comment:
Why Argument() singular and not Arguments() plural?

Reply:
Because Arguments imply its a vector. Argument implies a singleton.

Reply:
API Review feedback ;)

Reply:
Also aligns with what we have in Windows SDK today:
    [marshaling_behavior(standard)]
    [contract(Windows.Foundation.UniversalApiContract, 1)]
    runtimeclass ToastNotificationActionTriggerDetail
    {
        [contract(Windows.Foundation.UniversalApiContract, 1)]
        [interface_name("Windows.UI.Notifications.IToastNotificationActionTriggerDetail", 9445135a-38f3-42f6-96aa-7955b0f03da2)]
        {
            String Argument{ get; };
            Windows.Foundation.Collections.ValueSet UserInput{ get; };
        }
    }